src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1253,12 +1278,26 @@ private void InheritNullableStateOfTrackableStruct(TypeSymbol targetType, int ta
             }
         }
 
+        private bool IsSlotMember(int slot, Symbol possibleMember)
+        {
+            TypeSymbol possibleBase = possibleMember.ContainingType;
+            TypeSymbol possibleDerived = NominalSlotType(slot);
+            HashSet<DiagnosticInfo> discardedUseSiteDiagnostics = null;
+            var conversionsWithoutNullability = this.compilation.Conversions;
```

Comment:
>this.compilation.Conversions [](start = 48, length = 28)

For reference, this is also available (for essentially zero cost) from `_conversions.WithNullability(false)`.

Reply:
Sure, I'll switch to that and fix the concurrency bug in the implementation of `WithNullability` so that it doesn't fail an assertion.

---
In reply to: [366642586](https://github.com/dotnet/roslyn/pull/39301#discussion_r366642586) [](ancestors = 366642586)